src/routing/paths.ts

```diff
@@ -1,12 +1,19 @@
-export type PathString = `/${string}`
+import { type RelativePath } from '../types/navigation'
 
 interface Paths {
```

Comment:
I don't like this interface. I wanted it to be able to have arbitrary keys that were `RelativePath`s and a `dashboard` key that was `Paths`. However, TypeScript does not make this easy so I reluctantly went with a more rigid type.